libs/server/src/game/server_abstract_participant.cpp

```diff
@@ -1,15 +1,18 @@
-#include "server_abstract_participant.h"
-
-#include "../../color.h"
-#include "../../deck_list.h"
-#include "../../deck_list_card_node.h"
-#include "../../get_pb_extension.h"
-#include "../../rng_abstract.h"
-#include "../../trice_limits.h"
-#include "../server.h"
-#include "../server_abstractuserinterface.h"
-#include "../server_database_interface.h"
-#include "../server_room.h"
+#include "../../include/server/game/server_abstract_participant.h"
+
+#include "../../include/server/game/server_arrow.h"
```

Comment:
aren't these headers in the same location as this file?

Reply:
I see they are all in a separate include folder now, I'm not very enthusiastic about this move, but we could at the very least include it right? so we don't need all these ../..?